spring-cloud-zuul-ratelimit-tests/bucket4j-infinispan/src/main/java/com/marcosbarbero/tests/Bucket4jInfinispanApplication.java

```diff
@@ -0,0 +1,73 @@
+package com.marcosbarbero.tests;
+
+import io.github.bucket4j.grid.GridBucketState;
+import org.infinispan.AdvancedCache;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.functional.FunctionalMap.ReadWriteMap;
+import org.infinispan.functional.impl.FunctionalMapImpl;
+import org.infinispan.functional.impl.ReadWriteMapImpl;
+import org.infinispan.manager.DefaultCacheManager;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.SpringApplication;
+import org.springframework.cloud.client.SpringCloudApplication;
+import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Liel Chayoun
+ * @since 2018-04-07
+ */
+@EnableZuulProxy
+@SpringCloudApplication
+public class Bucket4jInfinispanApplication {
+
+    public static void main(String... args) {
+        SpringApplication.run(Bucket4jInfinispanApplication.class, args);
+    }
+
+    @Bean
+    @Qualifier("RateLimit")
+    public ReadWriteMap<String, GridBucketState> map() {
+        DefaultCacheManager cacheManager = new DefaultCacheManager();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "DefaultCacheManager" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)